primitives/src/ceremonies.rs

```diff
@@ -347,6 +348,22 @@ pub struct ReputationCacheValue {
 	pub reputation: Vec<(CeremonyIndexType, CommunityReputation)>,
 }
 
+#[derive(Encode, Decode, Copy, Clone, PartialEq, Eq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+#[cfg_attr(feature = "serde_derive", derive(Serialize, Deserialize))]
+pub enum MeetupResult {
+	Ok,
+	VotesNotDependable,
+	MeetupValidationIndexOutOfBounds,
+	NoResult,
+	Other,
```

Comment:
If these two are not used, I would remove them. `NoResult` I would never use. If this is semantically correct somewhere, I would rather go with an `Option<MeetupResult>` and then set it to `None`.